functions.sh

```diff
@@ -181,13 +181,13 @@ eerrorn()
 #
 eerror()
 {
-	if yesno "${EINFO_QUIET}"; then
-		printf " $*\n" >/dev/stderr
+	if yesno "${EERROR_QUIET}"; then
+		return 0
 	else
 		if ! yesno "${RC_ENDCOL}" && [ "${LAST_E_CMD}" = "ebegin" ]; then
-			printf "\n"
+			printf "\n" >&2
```

Comment:
After reading libeinfo a bit more, it looks like this newline would be written to stderr. I have confirmed this using strace.

So my code matches the behavior of the existing openrc code.